bin/cml/comment/create.js

```diff
@@ -55,5 +55,12 @@ exports.options = kebabcaseKeys({
     type: 'boolean',
     description:
       'Avoid watermark; CML needs a watermark to be able to distinguish CML comments from others'
+  },
+  commentLabel: {
+    type: 'string',
+    description:
+      'Label to add to watermarks for more selective comment updates',
```

Comment:
mention `{workflow,run}` template vars?

```suggestion
      'Comment identifier, where {workflow} and {run} are auto-replaced',
```